docs/tutorials/python/profiler.md

```diff
@@ -244,15 +239,17 @@ class CustomAddOneProp(mx.operator.CustomOpProp):
 
 inp = mx.nd.zeros(shape=(500, 500))
 
-profiler.set_config(profile_all=True, continuous_dump = True)
+profiler.set_config(profile_all=True, continuous_dump=True, \
+                   aggregate_stats=True)
 profiler.set_state('run')
 
 w = nd.Custom(inp, op_type="MyAddOne")
 
 mx.nd.waitall()
 
 profiler.set_state('stop')
-profiler.dump()
+print(profiler.dumps())
+profiler.dump(finished=False)
```

Comment:
As mentioned above, for this tutorial and majority of usecase, finished=True?

Reply:
This is because of the same issue as in the first comment